autopush/websocket.py

```diff
@@ -535,7 +556,8 @@ def process_ack(self, data):
         if not updates or not isinstance(updates, list):
             return
 
-        self.metrics.increment("updates.client.ack", tags=self.base_tags)
+        self.metrics.increment("updates.client.ack",
+                               tags=self._base_tags)
```

Comment:
Why isn't this using self.base_tags for the check it needs to ensure it doesn't use [] ?